### PR TITLE
feat(jasen) Add jasen workflow without subcommands

### PR DESCRIPTION
**Pull request by samuell**
### Background

[Jasen](https://github.com/genomic-medicine-sweden/jasen) is the new GMS-developed pipeline for bacterial typing and resistance determination.

It also has a repo at: https://github.com/clinical-genomics/mtp-jasen

The project for integrating Jasen is available here: https://github.com/Clinical-Genomics/project-planning/issues/533

### Added

- Added Jasen as workflow without any subcommands.

### How to prepare for test

- [x] Ssh to relevant server (depending on type of change)
- [x] Use stage: `us`
- [x] Paxa the environment: `paxa`
- [x] Install on stage (example for Hasta):
    ```shell
    bash /home/proj/production/servers/resources/hasta.scilifelab.se/update-tool-stage.sh -e S_cg -t cg -b [THIS-BRANCH-NAME] -a
    ```

### How to test

- [x] Run `cg workflow --help`

image

- [x] Run `cg workflow jasen`

image

- [x] Run
  ```
  pytest tests/cli/workflow/nf_analysis/test_cli_workflow_base.py
  ```
image

### Expected test outcome

- [x] `cg workflow --help` includes `jasen` in the list.
- [x] `cg workflow jasen` prints help.
  - Some warnings printed here, though seems unrelated to my code.
- [x] Take a screenshot and attach or copy/paste the output.

## Review

- [x] Tests executed by @samuell 
- [ ] "Merge and deploy" approved by

Thanks for filling in who performed the code review and the test!

### This [version](https://semver.org/) is a

- [ ] **MAJOR** - when you make incompatible API changes
- [x] **MINOR** - when you add functionality in a backwards compatible manner
- [ ] **PATCH** - when you make backwards compatible bug fixes or documentation/instructions

## Implementation Plan

- [ ] Document in ...
- [ ] Deploy this branch on ...
- [ ] Inform to ...

### CODE REVIEW

File: cg/cli/workflow/jasen/__init__.py [hunk 1]
```diff
@@ -0,0 +1 @@
+"""Init for module"""
```
Comment (diitaz93) [inline cg/cli/workflow/jasen/__init__.py:1]: I think this docstring is not necessary

File: cg/meta/workflow/jasen.py [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+LOG = logging.getLogger(__name__)
+
+
+class JasenAnalysisAPI(NfAnalysisAPI):
+    """Handles communication between Jasen processes and the rest of CG infrastructure."""
+
+    def __init__(
+        self,
+        config: CGConfig,
+        workflow: Workflow = Workflow.JASEN,
```
Comment (diitaz93) [inline cg/meta/workflow/jasen.py:18]: does this need to be a parameter since it will always be `Workflow.JASEN`?

**Review by diitaz93 — APPROVED** [review]
Looks good 👍

File: cg/meta/workflow/jasen.py (hunk 2 above)
Comment (samuell) [reply cg/meta/workflow/jasen.py:18]: I see that all the other workflows do this, and was assuming it is to conform to a common interface?

File: cg/cli/workflow/jasen/__init__.py (hunk 1 above)
Comment (samuell) [reply cg/cli/workflow/jasen/__init__.py:1]: Good point, although I have a sense that empty files can sometimes cause some troubles. Should we have a standard for this? :)

**Pull request merged**